CMake/activate_venv.sh.in

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# Script to activate a python virtual environment
+# and compose the differences in env variables
+# to setable env vars in cmake or the general environment
+export OLDPATH=$PATH
+conda activate @VENV_DIR@
+echo ${PATH%"$OLDPATH"} > conda_venv_path.txt
```

Comment:
I don't understand the syntax here:
echo ${PATH%"$OLDPATH"} > conda_venv_path.txt

What is the % doing?

Also, i don't see these scripts being generated anywhere. Are they supposed to turn into a .sh/.bat file somewhere?

Reply:
The % is the bash character for remove suffix, which here is removing all entries to the path prior to virtualenv activation from the current path, leaving the location of the python executable in the virtualenv. 

These scripts are generated here: [here](https://github.com/Kitware/kwiver/pull/1076/files#diff-cdbd9b26619cb98e22c4253e99db29ab0757b328f44b005543d8c01aea6700d7R5)

Reply:
Ah, very cool! I did not know that, thanks! Regarding the script generation, it looks like I need to satisfy `if (CONDA)` before they will generate, which requires `if (Python3_INTERPRETER_ID STREQUAL "Anaconda")`. If that's correct then I guess that would explain why I'm not seeing them.

Reply:
Right, we only need to use the scripts when we encounter conda, as the directory in which the virtualenvs are created can differ per installation, as opposed to the python module virtualenv, where the path can be explicitly specified upon creation. We run the scipts to capture the changes `conda activate env` makes to the path (and other env variables) so we can propagate them to our current CMake process context.